force-app/main/default/lwc/mineBestillingerWrapper/mineBestillingerWrapper.js

```diff
@@ -422,6 +465,7 @@ export default class MineBestillingerWrapper extends NavigationMixin(LightningEl
         this.hasFiles = event.detail > 0;
     }
 
+    // Ideally should show files immediately when uploaded. Need to run refreshApex on a query of ContentDocuments?
```

Comment:
Refresh apex wrapped in a then() method after upload file apex method should work yes